services/movie.py

```diff
@@ -0,0 +1,36 @@
+from django.db.models import QuerySet
+from db.models import Movie
+
+
+def get_movies(
+    genres_ids: list[int] = None, actors_ids: list[int] = None
+) -> QuerySet:
+    if genres_ids and actors_ids:
+        return Movie.objects.filter(
+            actors__in=actors_ids, genres__in=genres_ids
+        )
```

Comment:
The filter condition using `actors__in` and `genres__in` may result in unexpected behavior if a movie is associated with multiple actors or genres. Consider using `distinct()` to ensure unique results.